### ISSUE TITLE
AttributeError on regular style transfer without semantic annotations

### ISSUE THREAD

**Issue by carlthome** [alexjc/neural-doodle#43]
In the instructions it says _"To perform regular style transfer without semantic annotations, simply delete or rename the files with the semantic maps."_ but when I try to run a sample after having deleted both *_sem.png files I get an AttributeError error.

```bash
~/neural-doodle$ python3 doodle.py --style samples/Gogh.jpg --content samples/Seth.jpg --output output.png --device=gpu0 --iterations=80
Neural Doodle for semantic style transfer.
  - Using device `gpu0` for processing the images.
  - Loading content image data from samples/Seth.jpg.
  - Loading style image data from samples/Gogh.jpg.
Traceback (most recent call last):
  File "doodle.py", line 502, in <module>
    generator = NeuralGenerator()
  File "doodle.py", line 240, in __init__
    if self.content_map_original.shape[2] != self.style_map_original.shape[2]:
AttributeError: 'NoneType' object has no attribute 'shape'
```

The images are the same resolution. In fact the error appears even if the style and content image is the same file. I've successfully run the samples with the semantic annotations. What gives?

**Comment by alexjc** [comment]
I think I just fixed with [a660f1f]. Did you have that version when you tested?

**Comment by carlthome** [comment]
Yeah, working now. :+1:
